docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -268,11 +271,19 @@ kube-dns-{network} shall be managed by Endpoints controller just like a regular
 
 __EndpointSlices__, the new type introduced in k8s v1.17, is out of current scope.
 
-
-
 ### Ingress/egress/network policies
 (**TBD: more details required**)
 
+## CNI spec
+CNI_ARGS is leveraged for Arktos workload runtime to pass information on to the cni plugin of the network type.
+
+By spec the value of CNI_ARGS env var is alphanumeric key-value pairs separated by semicolons, e.g. "IPV4=12.34.56.78;VPCID=mizar-12345".
+
+Arktos workload runtime will put whatever is the pod annotation of "arktos.futurewei.com/cni-args" to CNI_ARGS as defined in cni spec.
+
+The network controller of the vpc network type is typically responsible for annotating pods properly with the relevant information.
+
+The alternative is 
```

Comment:
What are other alternatives?

Reply:
this incomplete line should be removed, as there is no good alternative yet.